.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "xz2-sys/xz"]
 	path = xz2-sys/xz
 	url = http://git.tukaani.org/xz.git
```

Comment:
@alexcrichton Want me to get rid of this submodule entry? It doesn't do anything since  the submodule directory isn't checked-in anymore.